app/src/interfaces/translations/translations.vue

```diff
@@ -250,14 +254,9 @@ function useLanguages() {
 			languages.value = response.data.data ? toArray(response.data.data) : [];
 
 			if (!firstLang.value) {
-				const userLang = languages.value.find(
-					(lang) =>
-						userStore.currentUser &&
-						'language' in userStore.currentUser &&
-						lang[pkField] === userStore.currentUser.language
-				)?.[pkField];
-
-				firstLang.value = userLang || languages.value[0]?.[pkField];
+				const locale = userLanguage.value ? i18n.global.locale.value : defaultLanguage.value;
```

Comment:
You can get the current locale using the `useI18n()` composable instead of using the global `i18n` object:

```js
const { t, locale } = useI18n();

// ...

const locale = userLanguage.value ? locale.value : defaultLanguage.value;
```